docs/readmes/contributing/contribute_conventions.md

```diff
@@ -0,0 +1,188 @@
+---
+id: contribute_conventions
+title: Code Conventions
+hide_title: true
+---
+
+# Code Conventions
```

Comment:
As you describe both code and documentation conventions would it be better to name this guide 'Code and Documentation Conventions?'